CalendarApp/BookingRoom/Controllers/UpdatingController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using BookingRoom.Models;
+using BookingRoom.Models.Posting;
+using Google.Apis.Calendar.v3.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+namespace BookingRoom.Controllers
+{
+    public class UpdatingController : ApiController
+    {
+        [HttpPut]
+        public HttpResponseMessage UpDateEvent(CalendarEvent eventForUpdating,string eventID)
+        {
+            CalendarConnection connection = new CalendarConnection();
+            Event upDatingEvent = new Event();
+            upDatingEvent.EventCopy(eventForUpdating);
```

Comment:
This is bad usage of extension method. I'd suggest to remove extension method and create a constructor that takes CalendarEvent instead